index.html

```diff
@@ -0,0 +1,179 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1" />
+    <title>KMarketer</title>
+    <link
+      href="https://fonts.googleapis.com/css?family=Montserrat"
+      rel="stylesheet"
+    />
+    <style>
+      body {
+        background-image: url(background.png);
+        background-repeat: no-repeat;
+        background-size: cover;
+        background-attachment: fixed;
+        position: relative;
+        margin: 0 auto;
+      }
+      body::before {
+        content: "";
+        display: block;
+        width: 100%;
+        height: 100%;
+        position: fixed;
+        top: 0;
+        left: 0;
+        background-color: rgba(0, 0, 0, 0.5);
+        z-index: -1;
+      }
+      .container {
+        display: flex;
+        max-width: 1050px;
+        width: 860px;
+        flex-direction: column;
+        align-items: center;
+        margin: 0 auto;
+        padding: 40px 0px;
+        gap: 40px;
+      }
+      h1 {
```

Comment:
Avoid using tag names as selectors (unless you have no other choice). Better use classes. And since your headings have many common styles you can use complex selector `h1, h4 {common style...}` same as you did with buttons